pkgs/by-name/kc/kcl/package.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "kcl";
+  version = "0.8.4";
+
+  src = if stdenv.isDarwin && stdenv.isAarch64 then fetchurl {
```

Comment:
When having this many sources, it might be advisable to create a `sources.json` and then read the URL endings and hashes from there. For reference, see e.g. the `tabnine` package.